src/components/PreviewComment/index.js

```diff
@@ -132,6 +132,11 @@ export const styles = StyleSheet.create({
     color: COLORS.gray410,
     marginLeft: SIZES.base
   },
+  profileContainer: {
+    flexDirection: 'column',
+    display: 'flex',
+    flex: 1
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

As mentioned above, the `display: 'flex'` property is not necessary in React Native styling. The `flexDirection` and `flex` properties already imply a flex container.
```diff
  profileContainer: {
    flexDirection: 'column',
-    display: 'flex',
    flex: 1
  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->